### PR TITLE
[BUGFIX] Add missing elements to node import service

### PR DESCRIPTION
**Pull request by thomashempel**
This is related to NEOS-1554 - I'm not sure if this fixes the entire issue.

### CODE REVIEW

**Comment by hlubek** [comment]
:+1: by reading, I'd suggest to add a new test case to `TYPO3\TYPO3CR\Tests\Unit\Domain\Service\ImportExport\NodeImportServiceTest` to cover these properties

File: TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php [hunk 1]
```diff
@@ -512,6 +512,9 @@ protected function persistEntities($propertyValue) {
 	 */
 	protected function parseEndElement(\XMLReader $reader) {
 		switch ($reader->name) {
+			case 'hiddenBeforeDateTime':
+			case 'hiddenAfterDateTime':
+			case 'contentObjectProxy':
```
Comment (aertmann) [inline TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php]: why contentObjectProxy? it's not present in parseElement, so should throw an exception if present.. I would remove it and create a separate change

**Comment by aertmann** [comment]
Hi Thomas

Thanks for pushing this one.

I agree with Christopher, this should have a test case. I did the same when introducing the creation & lastModification dates, so should be fairly straight forward.

Additionally this PR must be made for the lowest branch, so please close this one and push one for the 1.2 branch.

Third, it would be good practice to create a ticket for referencing that others can also find.

**Comment by thomashempel** [comment]
Hi, as Aske requested, I will close this one and provide a testcase and a patch for 1.2 branch

File: TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php (hunk 1 above)
Comment (thomashempel) [inline TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Service/ImportExport/NodeImportService.php]: I'm actually not sure about that one. I just checked in the export service what element names are created and added that. Not sure what it does though. I will keep that out in the next patch.

**Pull request closed without merging**